includes/e2k/processor-platform.h

```diff
@@ -0,0 +1,74 @@
+/*
+ *    HardInfo - Displays System Information
+ *    Copyright (C) 2003-2006 Leandro A. F. Pereira <leandro@hardinfo.org>
```

Comment:
I didn't write this.  You can put your name here instead.

Reply:
Fixed